lib/GenericsAPI/Utils/MatrixUtil.py

```diff
@@ -2187,24 +2149,35 @@ def perform_variable_stats_matrix(self, params):
 
     def transform_matrix(self, params):
         """
-        transform a matrix
+        Transform a matrix
         """
 
+        OPS = [
+            'abundance_filtering',
+            'relative_abundance',
+            'standardization',
+            'ratio_transformation',
+            'log',
+            'sqrt',
+            'logit',
+        ]
+
         logging.info('Start performing transformation with {}'.format(params))
 
         input_matrix_ref = params.get('input_matrix_ref')
-        workspace_name = params.get('workspace_name')
+        workspace_id = params.get('workspace_id')
         new_matrix_name = params.get('new_matrix_name')
 
-        abundance_filtering_params = params.get('abundance_filtering_params')
+        operations = params.get('operations')
+        abundance_filtering_params = params.get('abundance_filtering_params', {})
         perform_relative_abundance = params.get('perform_relative_abundance', False)
```

Comment:
remove the unused variable? not a big deal. But.

Reply:
For this, why is it always done on the column? Should we parameterize it with col/row? I probably should have raised this issue earlier

Reply:
I wonder if we should introduce a transpose operation. Just a brainstorm idea

Reply:
Would mean less row/col parameters to deal with

Reply:
I guess that would be less convenient for the amplicon workflow. But a more generic name for "relative abundance" is probably "to compositional" or something. Just another low-priority idea

Reply:
we should probably have row/col for `relative_abundance`. I think I didn't add row option because it wouldn't make sense for amplicon data. But for generic matrix, we should have that option. I will add the row option once this PR is merged. I am not sure how can we handle the transpose matrix. Because col and row are attached to samples and amplicon set. I guess at the very last step we can count how many transposes have been done and restore the original order.